docarray/index/backends/elastic.py

```diff
@@ -139,8 +152,14 @@ def __init__(self, outer_instance, **kwargs):
                 'query': defaultdict(lambda: defaultdict(list))
             }
 
+            self._outer_instance._logger.debug('QueryBuilder is being initialized')
```

Comment:
in a first time lets not add logs to the query builder

Reply:
what do we mean by first time?